addons/l10n_fr/__manifest__.py

```diff
@@ -1,51 +1,22 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
-    'name': 'France - Accounting',
-    'website': 'https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/france.html',
+    'name': 'France - Localizations',
     'icon': '/account/static/description/l10n.png',
```

Comment:
Why still keep l10n_fr?  Just define the little fields on company and partner in both modules.